pkg_defs/Ska.tdb/meta.yaml

```diff
@@ -4,6 +4,7 @@ package:
 
 build:
   noarch: python
+  preserve_egg_dir: yes
```

Comment:
I don't understand this commit just now.  With namespace packages it needs to reverted, no?

Reply:
1) I did the change before finishing our discussion in the other thread
2) With namespace packages they all can be set to remove the egg_dir directive and the --egg pip command I think.   But this is the only namespace package that was missing this in the current recipes and if for some reason we don't finish up the namespace stuff I didn't want to forget to fix it. 
3) So I think it gets moot in a hurry but doesn't cause any problems